derive_builder/tests/bounds_generation.rs

```diff
@@ -0,0 +1,55 @@
+#[macro_use]
+extern crate derive_builder;
+
+/// A struct that deliberately doesn't implement `Clone`.
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct Dolor(String);
+
+/// Notice that this type derives Builder without disallowing
+/// `Lorem<Dolor>`.
+#[derive(Debug, Clone, Builder, PartialEq, Eq)]
+#[builder(field(private), setter(into))]
```

Comment:
I know, we have deprecations all over the place. :-)

But we also run all tests a second time with `feature = "private_fields"`. So I think it's best not to add `#[builder(fields(private))]` on every test, but rather live with the deprecations for one cycle.